website_cookie_notice/__openerp__.py

```diff
@@ -21,20 +21,19 @@
 {
     'name': 'Cookie notice',
     'summary': 'Show cookie notice according to cookie law',
-    'version': '8.0.1.0.0',
+    'version': '9.0.2.1.0',
```

Comment:
For new major version, you should reset totally the rest of the numbers, as we  "start again", so the correct is 9.0.1.0.0.